user-service/src/component-tests/java/uk/gov/gchq/palisade/component/user/service/UserServiceAsyncProxyErrorTest.java

```diff
@@ -55,64 +56,61 @@
 
```

Comment:
Can't comment on the correct line (L51), but is this not just a `UserServiceAsyncProxyTest` as it tests successes and errors?

Reply:
updated